include/fem-dem/gls_sharp_navier_stokes.h

```diff
@@ -543,10 +543,11 @@ Return a bool that describes  if a cell contains a specific point
     vertices_to_cell;
   /*
    * This map uses the cell as the key, and store the following information:
-   * if that cell is cut (bool) and what particle cut this cell (unsigned int).
+   * if that cell is cut (bool), what particle cut this cell (unsigned int), and
+   * the number of particles that cut this cell.
    */
   std::map<typename DoFHandler<dim>::active_cell_iterator,
-           std::tuple<bool, unsigned int>>
+           std::tuple<bool, unsigned int, unsigned int>>
```

Comment:
I don't understand why you now have multiple particles that cut a cell, but you onlystore one particle id. What is that particle id? is it the id of the first prticle? Please be more clear :)